src/stats/timeline.R

```diff
@@ -40,6 +40,7 @@ result <- dbFetch(qry)
 dbClearResult(qry)
 dbCommit(rebenchdb)
 
+result <- cast_unnest_array(result) 
```

Comment:
Could you name this to be more specific?

How about `unnest_array_of_measurement_values`?